CLAUDE.md

```
@@ -157,6 +157,12 @@ The import hierarchy must be respected to prevent circular dependencies:
 - ESLint warnings should be fixed before PRs
 - Run `yarn run lint` to check for and fix ESLint issues
 
+### Comments and Documentation
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)